response_operations_ui/views/messages.py

```diff
@@ -180,9 +181,9 @@ def _thread_refine(message):
         'username': message.get('@msg_from').get('emailAddress'),
         # TODO use survey ref instead of survey id
         'survey': message.get('survey'),
-        'ruref': message.get('@ru_id').get('sampleUnitRef'),
-        'business': message.get('@ru_id').get('name'),
+        'ru_ref': message.get('@ru_id').get('sampleUnitRef'),
+        'business_name': message.get('@ru_id').get('name'),
         'to': _get_to_name(message),
-        'sentDate': message.get('sent_date'),
+        'sent_date': message.get('sent_date'),
         'internal': message.get('from_internal')
```

Comment:
Do we need a separate refine method, or can we just add the extra fields to `_refine`?

Reply:
all in one refine now